mongodb/kubedb_client_builder.go

```diff
@@ -152,6 +158,10 @@ func (o *KubeDBClientBuilder) getMongoDBClientOpts() (*mgoptions.ClientOptions,
 	if o.repSetName != "" {
 		repSetConfig = "replicaSet=" + o.repSetName + "&"
 	}
+	authDatabaseConfig := ""
```

Comment:
& this one `authDatabase`.  `authDatabaseConfig` portrays wrong information, this is not config database always.